### ISSUE TITLE
Model training

### ISSUE THREAD

**Issue by hendrikpesch** [linhdvu14/vggvox-speaker-identification#1]
Hi,

I saw that you are using Keras to replicate the model of the VoxCeleb paper. Did you train the model that is part of this repository yourself?
If yes, could you replicate the results from the paper, i.e. identification accuracy of 80%. I'm currently having trouble replicating this, because of overfitting. Any help would be appreciated.

Thanks!

**Comment by linhdvu14** [comment]
Hi,

The pretrained model is ported directly from the [official repo](https://github.com/a-nagrani/VGGVox) in Matlab.

The project currently only has code for evaluation. I have no immediate plan to implement the training part. Sorry about that.

You might want to check out the [voxceleb recipe from Kaldi](https://github.com/kaldi-asr/kaldi/tree/master/egs/voxceleb). It has code for training, testing, and several pretrained models.
